rpc/jeth.go

```diff
@@ -247,3 +248,76 @@ func (self *Jeth) confirmTransaction(id interface{}, jsonrpc string, args []inte
 	// Accept all tx which are send from this console
 	return self.client.Send(shared.NewRpcResponse(id, jsonrpc, true, nil)) == nil
 }
+
+// Sleep will halt the console for arg[0] seconds.
+func (self *Jeth) Sleep(call otto.FunctionCall) (response otto.Value) {
+	if len(call.ArgumentList) >= 1 {
+		if call.Argument(0).IsNumber() {
+			sleep, _ := call.Argument(0).ToInteger()
+			time.Sleep(time.Duration(sleep) * time.Second)
+			return otto.TrueValue()
+		}
+	}
+
+	fmt.Println("usage: sleep(<sleep in seconds>)")
```

Comment:
@fjl, didn't know that, cool feature. One snag, you actually have to panic on an otto value for this to work.

Updated code.